sabnzbd/config.py

```diff
@@ -435,10 +438,17 @@ def set_dict(self, values: Dict[str, Any]):
             "enable",
             "optional",
             "retention",
+            "expire_date",
+            "quota_left",
+            "usage_at_start",
             "priority",
             "notes",
         ):
             try:
+                if kw == "usage_at_start":
+                    usage_at_start_new = values.get("usage_at_start_new", "")
+                    if usage_at_start_new and values["quota_left"] != values["quota_left_old"]:
```

Comment:
I'm not sure I understand what is happening here?

Reply:
When the user changes the value (quota_left), the starting point for the usage calculation is changed to the current number of used bytes for that server and stored in usage_at_start. That way it's always relative to when the user added the quota and the quota doesn't have to be updated every time new bytes are downloaded.

Reply:
Aah now I understand 👍